projects/demo-meta-tedge.yaml

```diff
@@ -0,0 +1,53 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/siemens/kas/66261547b75885786777a0b9c8a4400ab81d432e/kas/schema-kas.json
+header:
+  version: 14
+  includes:
+    - config/common.yaml
+    - config/raspberrypi.yaml
+    - config/mender-raspberrypi.yaml
+    - demo-meta-tedge.lock.yaml
```

Comment:
You shouldn't reference the lock file in the includes (or at least I haven't found any docs that suggest you should do this, though correct me if I'm wrong)

Reply:
The docs say that the lock file should just be located "next to" the yaml file on disk. https://kas.readthedocs.io/en/latest/userguide.html#working-with-lockfiles

Reply:
Well in my case If I don't include lock file, It won't actually work (the commits were different during build than provided by the lock file)

Reply:
I found the root cause...the build-project just task was using the `--update` flag which causes the lock file detection mechanism to be ignored (hence why manually including it was required for it to work).

I've created a PR to avoid using the `--update` option by default. https://github.com/thin-edge/meta-tedge-project/pull/13